src/Akka.HealthCheck.Hosting.Web/README.md

```diff
@@ -0,0 +1,181 @@
+# Akka.HealthCheck.Hosting.Web
```

Comment:
Make sure this README is what ends up inside the `Akka.HealthCheck.Hosting.Web` NuGet package when it's pushed. Include some of the other boilerplate about Akka.NET and Akka.Hosting that a user might need to know too.